chainer/functions/activation/softmax.py

```diff
@@ -36,17 +37,17 @@ def forward(self, x):
             one = numpy.array(1, dtype=oz_dtype).ctypes
             zero = numpy.array(0, dtype=oz_dtype).ctypes
             handle = cudnn.get_handle()
-            x_cube = x[0].reshape(x[0].shape[:2] + (-1, 1))
+            x_cube = x[0].reshape(self._get_cube_shape(x[0].shape))
             desc = cudnn.create_tensor_descriptor(x_cube)
             self.y = xp.empty_like(x[0])
             libcudnn.softmaxForward(
                 handle, _algorithm, _mode, one.data, desc.value,
                 x_cube.data.ptr, zero.data, desc.value,
                 self.y.data.ptr)
         else:
```

Comment:
|Host|CUDA|CUDNN|Python|Test|
|---|---|---|---|---|
|jenkins|6.5|2.0|2.7|NG|
|jenkins|7.0|4.0|3|OK|
|jenkins|7.5|5.1|3.5|OK|
|my local|8.0|5.1|2.7|OK|
|my local|8.0|5.1|3.5|OK|

The error that you are mentioning might be dependent on the CUDA/CUDNN version.
I'd like to see if using 4D tensors will resolve the error but couldn't figure out a way to trigger the build...